recipes/r-lasr/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "lasr" %}
+{% set version = "0.13.7" %}
+{% set posix = "m2-" if win else "" %}
+
+package:
+  name: r-{{ name }}
```

Remove single-use variable.

```suggestion
{% set version = "0.13.7" %}
{% set posix = "m2-" if win else "" %}

package:
  name: r-lasr
```